applications/error-notifications.md

```diff
@@ -0,0 +1,32 @@
+# Error Notifications
+
+In order to alert teams to server errors for applications we need a way of
+posting these out from applications.  Currently we have development and
+production instances of [Errbit](https://github.com/errbit/errbit), an
+open source product compatible with [airbrake](https://airbrake.io/)
+
+Details of these instances can be obtained from your local development lead.
+
+## NodeJS
+
+[node-airbrake](https://github.com/airbrake/node-airbrake) can be used for Node
+applications.  It has recently had an update for compatibility with HapiJS
+(it already had expressJS), however in order to load it correctly as a plugin
+or using glue and a manifest please see the HapiJS section below.
+
+### HapiJS
```

Comment:
Would it be fair to say [Winston](https://github.com/winstonjs/winston) is more about logging in general @sgardnerdell-ea (though I can see from the README it can differentiate between errors and just general logging)?

In which case I would suggest we get this PR in and done, and then it'd be awesome if you could add a 'Logging' section under 'Applications', and give us a bit of info about using Winston in NodeJS apps. Particularly as again from the README it seems you have multiple ways you can implement Winston, so knowing how you've done it would be a great start.